.github/workflows/nightly.yml

```diff
@@ -233,9 +63,8 @@ jobs:
           time=$(git show --no-patch --format=%ad --date=format:%H%M%S)
 
           # Update the version to a nightly one
-          # This sets the nightly version to something like `1.20231117.1130416`
-          # The 1 at the beginning of the patch number is just so it never starts with zero
-          sed -i "s#^version = \"\([[:digit:]]*\)\..*\"#version = \"\1.${date}.1${time}\"#" lib/Cargo.toml
+          # This sets the nightly version to something like `1.20231117.130416`
```

Comment:
Also the version here is now wrong. The patch must start with `11` like in the comment that was replaced by this one.

Reply:
I didn't include the changes in https://github.com/surrealdb/surrealdb/pull/3012, only what I had in `main`. Will fix!